zeus/client.py

```diff
@@ -237,7 +234,6 @@ def main_random_cast(voter_url_file, plaintexts_file, nr_threads=2):
     if exists(plaintexts_file):
         m = "%s: file exists, will not overwrite" % (plaintexts_file,)
         raise ValueError(m)
-    f = open(plaintexts_file, "w")
```

Comment:
Something is wrong here. Below we have:

```
     plaintexts = [outqueue.get() for _ in range(total)]
     f.write(repr(plaintexts))
     f.close()
```

So probably that `f` belongs down there, and this should be `with open(plaintexts_file, 'w') as f` etc.